### ISSUE TITLE
Add GNOME 44 Support

### ISSUE THREAD

**Issue by username0136** [BjoernDaase/noannoyance#28]

**Comment by staticf0x** [comment]
Yes please, Fedora 38 is now out with Gnome 44

**Comment by staticf0x** [comment]
By the way you can install it from extensions.gnome.org if you disable version validation:
![image](https://user-images.githubusercontent.com/44530786/233408965-66a0c243-0ecd-4344-bf36-e4c0af119b1b.png)

it works fine

**Comment by Dispose6066** [comment]
For those who decided to disable version validation, don't be surprised by https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/6507

**Comment by staticf0x** [comment]
Good point, I was a bit surprised, but restarting `gdm` fixes that meanwhile (that's two temporary fixes already lol)

**Comment by taisph** [comment]
There's a PR ready in #29.

**Comment by Flimm** [comment]
Ubuntu 23.04 has been released with GNOME 44.

**Comment by azlan** [comment]
Time to fork this project as NoAnnoyance v3 :smiley_cat:

**Comment by boozedog** [comment]
Is this project still supported?

**Comment by medovina** [comment]
Apparently not, since the pull request has been ready for weeks and nobody has merged it.  It would be great if someone would like to make a new fork, ideally someone who can commit to keeping it alive for at least a year or two.  :)  If anyone is willing and does so, please announce it here!

**Comment by xmsanchez** [comment]
You can still make it work just by updating the metadata.json file adding "44" to the array, I've done this a lot of times for other extensions and usually works fine between minor versions (Gnome 4x).

But yeah, ideally it should be maintained by someone.

**Comment by jirkavrba** [comment]
I've published my fork that adds support for GNOME 44 on the GNOME extensions portal.
You can install it by visiting the link below.

https://extensions.gnome.org/extension/6109/noannoyance-fork/

**Comment by pezz** [comment]
jirkavrba's fork doesn't work on GNOME 44 and there's no issues enabled on his repo.

**Comment by nose-gnome** [comment]
> jirkavrba's fork doesn't work on GNOME 44 and there's no issues enabled on his repo.

@jirkavrba Would it be possible for you to enable issues on your repo?
